lib/item.rb

```diff
@@ -0,0 +1,26 @@
+class Item
+
+  attr_reader :price
+
+  def initialize(name, price, reference)
+    @name = name
+    @price = price
+    @reference = reference
+  end
+
+  def name
+    @name.clone
+  end
+
+  def reference
+    @reference.clone
+  end
+
+end
+
+
+
+
+#could possibly be done as a struct but since if this were expanded there would probably be photos and other details like if an item is out of stock
```

Comment:
Metrics/LineLength: Line is too long. [148/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)